src/lab6.py

```diff
@@ -231,7 +231,7 @@ def flush(self):
         baseline = self.cursor_y + 1.25 * max_ascent
         for rel_x, word, font, color in self.line:
             x = self.x + rel_x
-            y = baseline - font.metrics("ascent")
+            y = self.y + baseline - font.metrics("ascent")
```

Comment:
Can you add a unittest? Also, does this draw the browser.engineering homepage wrong? I would think everything looks terrible without this bugfix.

Reply:
Yeah it looks like crap. I found this because I was debugging Safari and thought it was a Safari bug. then I checked Chrome, thought it was a compiler bug, eventually thought to try the Python. I can add a test.

Reply:
Added a test. Along the way, I removed two `__repr__` methods in lab6 that were desynced from lab5 and which made writing the test hard.

Reply:
(The test is just a copy-and-paste of a lab5 test)